racker/salus-telemetry-resource-management
# Adam9500/salus 318 swagger documentation

# Resolves

    SALUS-349

# What

updates the pom to pass replacement parameters into 

# How

Sends more parameters into the SwaggerJsonConverter

## How to test

compile then install.
Then checkout racker/salus-event-engine-management#4 and run tests. Then you will need to check the convertedOutput.json
